### DISCUSSION TITLE
Only some of the contents of Google Calendar are synchronized.

### DISCUSSION THREAD

**Discussion by einspannerisgood** [ivan-lednev/obsidian-day-planner#444] [Q&A]
Prior to the question, I use a translator.
I added Google Calendar as a sub as per the notice. 

My problem is
1. The schedule that has been deleted and does not exist is visible and cannot be deleted.
2. How can I solve a problem that only reflects my personal workout schedule?

I'd like to bring all the content of Google Calendar. Please comment!

**Comment by ivan-lednev** [comment]
Hello! 

> The schedule that has been deleted and does not exist is visible and cannot be deleted.

This is a bug, and I'm going to fix it in a future release.

> How can I solve a problem that only reflects my personal workout schedule?

Sorry, I didn't get that one, could you clarify a bit?
